src/bot.py

```diff
@@ -8,8 +8,8 @@
 import aiohttp
 from PIL import Image
 
-from models import Contributor, Organization
 from src import secrets
+from src.models import Contributor, Organization
```

Comment:
Was this really required?
It was running fine on my end both manually and in Docker.

Reply:
@diwash007 For some reason it raised an error in my end.